src/sidebar/frame-sync.js

```diff
@@ -174,6 +174,16 @@ function FrameSync($rootScope, $window, Discovery, annotationUI, bridge) {
         });
       }
 
+      // if doi available as highwire or dc meta, include in search
+
+      if ( info.metadata && info.metadata.highwire && info.metadata.highwire.doi ) {
+        searchUris.push('doi:' + info.metadata.highwire.doi[0]);
+      }
+
+      if ( info.metadata && info.metadata.dc && info.metadata.dc.doi)  {
+        searchUris.push('doi:' + info.metadata.dc.doi[0]);
```

Comment:
I note that there is already some code in document.coffee which populates the `info.metadata.link` Array with `doi:` URLs. We should only do DOI metadata => "doi:" URI extraction in one place, so we'll need to figure out whether that should happen here or in document.coffee.